### PR TITLE
Database setup

### PR DESCRIPTION
**Pull request by YannYves**

### CODE REVIEW

**Comment by YannYves** [comment]
This is the database setup V1

**Review by JulienMartos — DISMISSED** [review]

File: app.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// app.js
```
Comment (LoicBrassart) [inline app.js]: Why this comm?

File: app.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+const express = require("express");
+const bodyParser = require("body-parser");
+require("dotenv").config();
+
+const connection = require("./connection");
+const app = express();
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+
+//
```
Comment (LoicBrassart) [inline app.js]: Why this comm?

File: connection.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+// connection.js
```
Comment (LoicBrassart) [inline connection.js]: Why this comm?

File: routes/formAss.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const connection = require("../connection");
+const router = express.Router();
+
+// GET tous les form_assurance
+router.get("/all", (req, res) => {
+  connection.query(" SELECT * FROM form_assurance", (err, results) => {
```
Comment (LoicBrassart) [inline routes/formAss.js]: ("Select *" is a bad practice)

File: routes/formBanks.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const connection = require("../connection");
+const router = express.Router();
+
+// GET tous les form_banque
+router.get("/form/banque", (req, res) => {
+  connection.query(" SELECT * FROM form_banque", (err, results) => {
```
Comment (LoicBrassart) [inline routes/formBanks.js]: ("Select *" is a bad practice)

File: routes/partners.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const connection = require("../connection");
+const router = express.Router();
+
+// GET tous les partners
+router.get("/", (req, res) => {
+  connection.query(" SELECT * FROM partners", (err, results) => {
```
Comment (LoicBrassart) [inline routes/partners.js]: ("Select  * is a bad practice")

File: routes/users.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const connection = require("../connection");
+const router = express.Router();
+
+// GET tous les users
+router.get("/all", (req, res) => {
+  connection.query(" SELECT * FROM users", (err, results) => {
```
Comment (LoicBrassart) [inline routes/users.js]: ("Select * is a bad practice")

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Please correct some mistakes

File: routes/formAss.js [hunk 8]
```diff
@@ -4,13 +4,18 @@ const router = express.Router();
 
 // GET tous les form_assurance
 router.get("/all", (req, res) => {
-  connection.query(" SELECT * FROM form_assurance", (err, results) => {
-    if (err) {
-      res.status(500).send("Erreur lors de la récupération des données" + err);
-    } else {
-      res.status(200).json(results);
+  connection.query(
+    " SELECT patners_a patners_b FROM form_assurance",
```
Comment (LoicBrassart) [inline routes/formAss.js]: "patners" ? Are you sure ?

File: routes/formBanks.js [hunk 9]
```diff
@@ -3,14 +3,19 @@ const connection = require("../connection");
...
 // GET tous les form_banque
-router.get("/form/banque", (req, res) => {
-  connection.query(" SELECT * FROM form_banque", (err, results) => {
-    if (err) {
-      res.status(500).send("Erreur lors de la récupération des données" + err);
-    } else {
-      res.status(200).json(results);
+router.get("/all", (req, res) => {
+  connection.query(
+    " SELECT parrtners_a parrtners_b  FROM form_banque",
```
Comment (LoicBrassart) [inline routes/formBanks.js]: "parrtners" ? Are you sure ?

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Typos in requests
Did you test your work before you re-pushed ?

**Review by JulienMartos — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**